### ISSUE TITLE
Compile error "ncurses@v0.2.0/keys.go:171:2: could not determine kind of name for C.KEY_EVENT"

### ISSUE THREAD

**Issue by rpoe** [seehuhn/go-ncurses#3]
package is not compiling. 

Reason is ncurses.h does not define KEY_EVENT.

**Comment by seehuhn** [comment]
Dear Ralf,

The package still compiles for me, so this must be something system specific.  What version of the ncurses library are you using?  What operating system are you on?

Many thanks,
Jochen

**Comment by panicz** [comment]
I have the same problem. I'm trying to run it on Windows/WSL1, and I get exactly the same error message.
When I grep `/usr/include` for, say `KEY_BREAK`, I get entries in `/usr/include/curses.h` (which I think  might be the most relevant) as well as `/usr/include/linux/input-event-codes.h`. But grepping for `KEY_EVENT` gives no results.

**Comment by seehuhn** [comment]
In an attempt to fix this, I will simply comment out the KEY_EVENT line in the next version.  Hopefully this will help ...

**Comment by Erryo** [comment]
I encountered the same problem despite this issue being closed. It seems that the Go documentation and source files are outdated. Specifically, line 171 in keys.go was not commented out when I checked.

I got the package by running the usual `go get seehuhn.de/go/ncurses`

I fixed the problem  by commenting out line 171 at `/home/USERNAME/go/pkg/mod/seehuhn.de/go/ncurses@v0.2.0/keys.go`.I am leaving this comment here in case somebody needs it.
